lochness/box/__init__.py

```diff
@@ -325,34 +334,35 @@ def sync_module(Lochness: 'lochness.config',
                         continue
 
                     protect = product.get('protect', False)
-                    compress = product.get('compress', False)
-                    key = enc_key if protect else None
                     output_base = subject.protected_folder \
                                   if protect else subject.general_folder
 
-                    if 'processed' in root:
-                        processed = True
-                    else:
-                        processed = False
+                    encrypt = product.get('encrypt', False)
+                    key = enc_key if encrypt else None
 
-                    # output_base = tree.get(datatype, output_base)
+                    processed = product.get('processed', False)
```

Comment:
Why does the `processed` still exist? Did we not get rid of the notion of `raw` and `processed` at the remote end?

Reply:
Yes, we did remove this from our example structure. However, the functionality to take specifications from the config.yml is still available in the code for future projects in need of this function. I'll revisit the structure, when I work on centralizing the output directory soon.